docker/databricks/init-alluxio.sh

```diff
@@ -0,0 +1,230 @@
+    set_alluxio_property alluxio.worker.tieredstore.level0.dirs.path "${paths}"
+    set_alluxio_property alluxio.worker.tieredstore.level0.dirs.quota "${quotas}"
+  fi
+  echo "${use_mem}"
+}
+
+# add crontab to rsync alluxio log to /dbfs/cluster-logs/alluxio
+set_crontab_alluxio_log() {
+  if [[ "$#" -ne "1" ]]; then
+    echo "Incorrect"
+    exit 2
+  fi
+  local folder=$1
+  mkdir -p /dbfs/cluster-logs/alluxio/$folder
+  # add crond to copy alluxio logs
+  crontab -l > cron_bkp || true
+  echo "* * * * * /usr/bin/rsync -a /opt/alluxio-2.8.0/logs /dbfs/cluster-logs/alluxio/$folder >/dev/null 2>&1" >> cron_bkp
+  crontab cron_bkp
+  rm cron_bkp
+}
+
+
+# create the folder for NVMe caching
+mkdir -p /local_disk0/cache
+chown ubuntu:ubuntu /local_disk0/cache
+
+# create the folder for domain socket
+mkdir -p /local_disk0/alluxio_domain_socket
+chown ubuntu:ubuntu /local_disk0/alluxio_domain_socket
+
+set_alluxio_property alluxio.master.hostname "${DB_DRIVER_IP}"
+set_alluxio_property alluxio.underfs.s3.inherit.acl false
+set_alluxio_property alluxio.underfs.s3.default.mode 0755
+set_alluxio_property alluxio.worker.tieredstore.levels "1"
+set_alluxio_property alluxio.worker.data.server.domain.socket.address /local_disk0/alluxio_domain_socket
+set_alluxio_property alluxio.worker.data.server.domain.socket.as.uuid true
+#set_alluxio_property alluxio.worker.network.async.cache.manager.threads.max 16
+set_alluxio_property alluxio.worker.network.async.cache.manager.queue.max 4000
+set_alluxio_property alluxio.user.short.circuit.preferred true
+
+# set metrics properties
+#set_metrics_property sink.csv.class alluxio.metrics.sink.CsvSink
+#set_metrics_property sink.csv.period 5
+#set_metrics_property sink.csv.unit seconds
+#set_metrics_property sink.csv.directory /opt/alluxio-2.8.0/logs
+
+#set_alluxio_property alluxio.user.block.size.bytes.default 16M
+#set_alluxio_property alluxio.underfs.s3.threads.max 80
+
+#set_alluxio_property alluxio.job.worker.threadpool.size 30
+set_alluxio_property alluxio.job.worker.threadpool.size 20
+set_alluxio_property alluxio.worker.network.block.writer.threads.max 2048
+set_alluxio_property alluxio.worker.network.block.reader.threads.max 2048
+#set_alluxio_property alluxio.user.network.netty.worker.threads 30
+set_alluxio_property alluxio.master.ufs.block.location.cache.capacity 2000000
+set_alluxio_property alluxio.master.ufs.path.cache.capacity 200000
+
+#this default to number of cores on system, do we need to increase?
+#set_alluxio_property alluxio.master.metadata.sync.executor.pool.size
+#set_alluxio_property alluxio.master.rpc.executor.max.pool.size 1000
+
+sed -i "s/localhost/${DB_DRIVER_IP}/g" /opt/alluxio-${ALLUXIO_VERSION}/conf/masters
+
+configure_nvme "${ALLUXIO_STORAGE_PERCENT}"
+
+if [[ $DB_IS_DRIVER = "TRUE" ]]; then
+  # On Driver
+  set_crontab_alluxio_log "${DB_DRIVER_IP}-master"
+  doas ubuntu "ALLUXIO_MASTER_JAVA_OPTS='-Xms16g -Xmx16g' ${ALLUXIO_HOME}/bin/alluxio-start.sh master"
```

Comment:
memory size here should be configurable based on the node type.  I guess this isn't being called anywhere yet.  Ideally if we put this into docker image to do the base set of configs we have an easy way for user to set or override alluxio configs from the init script as well